.pipelines/ci/templates/build-powertoys-steps.yml

```diff
@@ -277,7 +243,6 @@ steps:
       **\KeyboardManagerEditorTest.dll
       **\UnitTests-CommonLib.dll
       **\PowerRenameUnitTests.dll
-      **\powerpreviewTest.dll
```

Comment:
Removed this since it doesn't appear to exist.

Also, I can probably get the Native Tests wired up as well which may save up to another minute, but scoping to just the managed tests for this PR.